pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/StatusAcknowledgeServlet.java

```diff
@@ -69,7 +70,7 @@ protected void doPost(@NotNull SlingHttpServletRequest request, @NotNull SlingHt
                 response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Locale other than en_US is not supported");
                 return;
             }
-            processAcknowledgementRequest(acknowledgement, module, moduleLocale);
+            processAcknowledgementRequest(acknowledgement, module, moduleLocale, request);
```

Comment:
Suggestion: Since only user id/name is needed in createStatusNode method, passing the name would be a better approach than passing the whole request objects. This would make other methods independent of request object and could make refactoring, if required in near future, simpler.

Reply:
AP, thank you for the suggestion. I've updated the code accordingly.